contrib/completion/bash/docker

```diff
@@ -1396,7 +1396,7 @@ _docker_network_ls() {
 
 	case "$prev" in
 		--filter|-f)
-			COMPREPLY=( $( compgen -S = -W "id name type" -- "$cur" ) )
+			COMPREPLY=( $( compgen -S = -W "driver id label name type" -- "$cur" ) )
```

Comment:
ping @albers I think this is where it needs to be added for bash completion

also @sdurrheimer I couldn't find the `--f` / `--filter` option for zsh-completion. I think it may need to be added here; https://github.com/docker/docker/blob/master/contrib/completion/zsh/_docker#L422-L426